src/vmm/src/config/mod.rs

```diff
@@ -362,6 +418,17 @@ mod tests {
         assert!(BlockConfig::try_from(block_str).is_err());
     }
 
+    #[test]
+    fn test_serial_config() {
```

Comment:
Should we add some tests with only the input set and not the output, and the output set, but not the input? A test case for an invalid setup would also be nice.